api/src/routes/donate.test.ts

```diff
@@ -0,0 +1,58 @@
+import request from 'supertest';
+
+import { setupServer, superRequest } from '../../jest.utils';
+
+describe('Donate', () => {
+  setupServer();
+
+  describe('Authenticated User', () => {
+    let setCookies: string[];
+
+    // Authenticate user
+    beforeAll(async () => {
+      const res = await request(fastifyTestInstance.server).get(
+        '/auth/dev-callback'
+      );
+      setCookies = res.get('Set-Cookie');
+    });
+
+    describe('POST /donate/add-donation', () => {
+      it('should return 200 and update the user', async () => {
+        const response = await superRequest('/donate/add-donation', {
+          method: 'POST',
+          setCookies
+        }).send({
+          isDonating: true
+        });
```

Comment:
this does not make sense to me.

Reply:
We should at least test if the endpoint works correctly.

Reply:
It does work correctly. It just doesn't currently care what the body is.

If you've got time, we can add some validation to the api-server. Then we can add that validation to the new api. Otherwise we can track that in a post-MVP issue.